code/modules/holiday/holidays.dm

```diff
@@ -502,7 +502,16 @@ Since Ramadan is an entire month that lasts 29.5 days on average, the start and
 	begin_month = DECEMBER
 	end_day = 27
 	drone_hat = /obj/item/clothing/head/santa
-
+	lobby_music = list(
+		"https://www.youtube.com/watch?v=v7s2VjwQSMw", // jingle bells
+		"https://www.youtube.com/watch?v=x2YlbiyiuMc", // Winter Wonderland
+		"https://www.youtube.com/watch?v=cW38y4AFGyI", // The Grinch
+		"https://www.youtube.com/watch?v=zgKazTrhXmI", // frosty the snowman
+		"https://www.youtube.com/watch?v=oyEyMjdD2uk", // twelve days of christmas
+		"https://www.youtube.com/watch?v=maYCStVzjDs",  // deer song
+		"https://www.youtube.com/watch?v=Dkq3LD-4pmM",  // jolly christmas
+		"https://www.youtube.com/watch?v=_9ePg09zV-0",  // i got a brand new dsi for christmas
```

Comment:
Remove this one and I'll merge it.

Reply:
which song?  you highlighted multiple

Reply:
the DSI song. Also, the last entry in the list shouldn't have a comma behind it.